src/styles/main.css

```
@@ -0,0 +1,322 @@
+  z-index: 1;
+  background-color: var(--color-bluegray-100);
+}
+
+.site-header--overlay {
+  background-color: transparent;
+}
+
+.site-header-inner {
+  position: relative;
+
+  display: flex;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)